app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardViewModel.kt

```diff
@@ -54,6 +63,15 @@ class ConceptCardViewModel @Inject constructor(
     return conceptCardResult.getOrDefault(ConceptCard.getDefaultInstance())
   }
 
+  private fun processConceptCardExplanation(conceptCardResult: AsyncResult<ConceptCard>): Spannable {
+    if (conceptCardResult.isFailure()) {
+      logger.e("ConceptCardFragment", "Failed to retrieve Concept Card: " + conceptCardResult.getErrorOrNull())
```

Comment:
Nit: please pass the error as an argument rather than concatenating it onto the error string.

Reply:
Updated to have error passed in as the third argument.